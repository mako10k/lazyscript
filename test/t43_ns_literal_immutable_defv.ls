!{
  ns <- { .Foo = 1; };
  ((~prelude nsdefv) ~ns .Bar 2)  # should error (immutable)
};
