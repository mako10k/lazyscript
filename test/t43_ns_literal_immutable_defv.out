E: #err "nsdefv: immutable namespace"
